automation_tools/__init__.py

```diff
@@ -556,6 +556,10 @@ def setup_oscap():
     for daemon in ('foreman', 'httpd', 'foreman-proxy'):
         manage_daemon('restart', daemon)
 
+def oscap_content():
```

Comment:
do we also need this function in fabfile.py if somebody wants to run it locally?